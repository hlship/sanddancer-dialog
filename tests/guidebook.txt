(setup lighting)
n
e
read book
%% *(understand [consult book about sand-dancer] as $)
consult book about sand-dancer 
consult book about lizards
consult book about pandas
consult book about rabbit 
look up coyote in book
consult book about coyotes


