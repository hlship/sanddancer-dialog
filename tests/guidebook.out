

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (setup lighting)
Query succeeded: (setup lighting)
> n
(first attempting to get out of the pickup truck)
The door creaks open with difficulty, rusted joints straining. You get out of
the pickup truck.

The cold night air swirls around you, and you pull your jacket closer.

You walk north.

Crumbling Concrete
This concrete building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> e
You walk east.

Weed Strewn Rust
Behind the rather small building is nothing but a scraggly patch of yellowing
weeds and dirt near a leaking pipe, littered by a collection of rusted barrels
and some scattered newspapers. Atop one overturned barrel is a weathered
guidebook.

Desert stretches in all directions; back west is the front of the building.

> read book
It's a guide book, you can CONSULT it about different desert creatures.

> consult book about sand-dancer 
You look up the brown-colored lizard in the guidebook and quickly identify it:
those eyes are a dead giveaway. The entry says it's a "rare subspecies of the
common desert lizard found only in the deep desert near Oro Oeste, and known to
native peoples as a sand-dancer. In legends, the sand-dancer was a clever
trickster and twister of words, who created night by tricking the sun into
spending half the day underground." Huh.

> consult book about lizards
You look up the brown-colored lizard in the guidebook and quickly identify it:
those eyes are a dead giveaway. The entry says it's a "rare subspecies of the
common desert lizard found only in the deep desert near Oro Oeste, and known to
native peoples as a sand-dancer. In legends, the sand-dancer was a clever
trickster and twister of words, who created night by tricking the sun into
spending half the day underground." Huh.

> consult book about pandas
There's a really odd mention of some pandas, escaped from a private menagerie,
thriving somewhere in the desert. You find that highly unlikely.

> consult book about rabbit 
You look up the hare in the guidebook. "Common desert rabbit," is says,
"frequently mistakent for a hare. Lives in underground burrows which it uses to
keep warm through the chill desert nights. Trusted by native peoples as a guide
and symbol of rebirth."

> look up coyote in book
You flip though the weathered guidebook but, unsurprisingly, find nothing about
that.

> consult book about coyotes
Weird, it looks like somebody's ripped out the page.

> 
